test/clangd_tests.vim

```diff
@@ -1691,7 +1691,7 @@ def g:Test_DiagVirtualText()
   g:LspOptionsSet({showDiagWithVirtualText: true})
   p = prop_list(1, {end_lnum: line('$')})
   assert_equal(1, p->len())
-  assert_equal([3, 8, 'LspDiagVirtualTextError'], [p[0].lnum, p[0].length, p[0].type])
+  assert_equal([3, 'LspDiagVirtualTextError'], [p[0].lnum, p[0].type])
```

Comment:
Before noticing this change in your PR, I already incorporated it in https://github.com/yegappan/lsp/commit/5e715cafa0f1652eff4bc171c98d949cbce6da67.  Can you update the PR?